### ISSUE TITLE
Document content is not saved when it was just created

### ISSUE THREAD

**Issue by jakobroehrl** [CollaboraOnline/richdocumentscode#238]
Maybe I found a bug. I only can reproduce it with Collabora Online - Built-in CODE Server (versions 23.5.301 and  23.5.202) but not with an AIO test installation or with collabora on docker.

- Create a new document (name, table or document, MS office or Open  Document does not matter)
- write some text and close it
- open it again, the file will be empty
- write some text again and close it
- open it again, the last text will be there

**Comment by thomaselsaesser** [comment]
I have the same effect. NC 27.0.2.1 and  latest richdocument version. If i save the document with save as docx , docx has the data, but the odt has no data. Only by first open.
Regards
Thomas

**Comment by mihail641** [comment]
I'm have same error. NC 27.0.2.1, richdocument 23.5.301. I'm write document after create about 4 hours. But nothing was saved!

**Comment by enrixxel** [comment]
same error here with nc 26.0.7 very annoying

**Comment by PeteBo** [comment]
I have the same issue with latest versions NC 27.1.4 and CODE 23.5.602 running on Ubuntu 22.04.3 PHP8.1

here is a snip of the coolwsd.log captured while creating a new odt file, writing a line of text ointo it, saving and closing it:

```
frk-03277-03277 2023-12-10 18:12:27.537906 +0100 [ forkit ] WRN  The systemplate directory [/tmp/.mount_CollabPx71Mb/] is read-only, and at least [/tmp/.mount_CollabPx71Mb//etc/passwd] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/tmp/.mount_CollabPx71Mb//etc] are up-to-date.| common/JailUtil.cpp:529
kit-409486-03277 2023-12-10 18:12:27.547947 +0100 [ kit_spare_030 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3045
frk-03277-03277 2023-12-10 18:12:27.573497 +0100 [ forkit ] WRN  The systemplate directory [/tmp/.mount_CollabPx71Mb/] is read-only, and at least [/tmp/.mount_CollabPx71Mb//etc/passwd] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/tmp/.mount_CollabPx71Mb//etc] are up-to-date.| common/JailUtil.cpp:529
frk-03277-03277 2023-12-10 18:12:27.580658 +0100 [ forkit ] WRN  The systemplate directory [/tmp/.mount_CollabPx71Mb/] is read-only, and at least [/tmp/.mount_CollabPx71Mb//etc/passwd] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/tmp/.mount_CollabPx71Mb//etc] are up-to-date.| common/JailUtil.cpp:529
kit-409492-03277 2023-12-10 18:12:27.581605 +0100 [ kit_spare_031 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3045
kit-409493-03277 2023-12-10 18:12:27.588474 +0100 [ kit_spare_032 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3045
frk-03277-03277 2023-12-10 18:12:42.299139 +0100 [ forkit ] WRN  The systemplate directory [/tmp/.mount_CollabPx71Mb/] is read-only, and at least [/tmp/.mount_CollabPx71Mb//etc/passwd] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/tmp/.mount_CollabPx71Mb//etc] are up-to-date.| common/JailUtil.cpp:529
kit-409643-03277 2023-12-10 18:12:42.306926 +0100 [ kit_spare_033 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3045
frk-03277-03277 2023-12-10 18:12:42.715867 +0100 [ forkit ] WRN  The systemplate directory [/tmp/.mount_CollabPx71Mb/] is read-only, and at least [/tmp/.mount_CollabPx71Mb//etc/passwd] is out-of-date. Will have to copy sysTemplate to jails. To restore optimal performance, make sure the files in [/tmp/.mount_CollabPx71Mb//etc] are up-to-date.| common/JailUtil.cpp:529
kit-409649-03277 2023-12-10 18:12:42.721791 +0100 [ kit_spare_034 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3045
wsd-03228-409648 2023-12-10 18:12:43.028829 +0100 [ docbroker_033 ] ERR  #-1: Serial mismatch 0 vs. 1| wsd/ProxyProtocol.cpp:168
wsd-03228-409648 2023-12-10 18:12:43.593975 +0100 [ docbroker_033 ] ERR  Expected to renamed the document [/tmp/coolwsd.39Djfdv2we/jails/3228-2d5d13ba/PhFtSnl2YNUZ0LWu/tmp/user/docs/PhFtSnl2YNUZ0LWu/SaveTest.odt.upload] after template-loading to [/tmp/coolwsd.39Djfdv2we/jails/3228-2d5d13ba/PhFtSnl2YNUZ0LWu/tmp/user/docs/PhFtSnl2YNUZ0LWu/SaveTest.odt.uploading] (ENOENT: No such file or directory)| wsd/DocumentBroker.cpp:1672
wsd-03228-409648 2023-12-10 18:13:08.293442 +0100 [ docbroker_033 ] ERR  #30: Read failed, have 0 buffered bytes (ECONNRESET: Connection reset by peer)| net/Socket.hpp:1139
```

after seeing the warnings and errors about the tmp folder i checked it's permissions and found this

```
root@vps1:~# ls -al /tmp/.mount_CollabPx71Mb
ls: cannot access '/tmp/.mount_CollabPx71Mb': Permission denied
root@vps1:~# ls -al /tmp
ls: cannot access '/tmp/.mount_CollabPx71Mb': Permission denied
total 4016
drwxrwxrwt 31 root          root            36864 Dec 10 18:20 .
drwxr-xr-x 20 root          root             4096 Dec  8 15:51 ..
drwx------  3 my-www-user   my-www-group     4096 Dec  7 15:50 coolwsd.39Djfdv2we
drwxrwxrwt  2 root          root             4096 Dec  7 15:47 .font-unix
drwxrwxrwt  2 root          root             4096 Dec  7 15:47 .ICE-unix
drwx------  2 my-www-user   my-www-group     4096 Dec  7 15:50 lu32773vvd.tmp
... more unrelated folders here ...
drwx------  2 my-www-user   my-www-group     4096 Dec  7 15:50 lu32773vvr.tmp
d?????????  ? ?             ?                   ?            ? .mount_CollabPx71Mb
-rw-------  1 my-www-user   my-www-group  3951456 Dec 10 14:10 nextcloud_photos_uLgnrK
... more unrelated folders here ...
```

**Comment by joshtrichards** [comment]
@PeteBo Something is not conventional about your `/tmp` folder/mountpoint from the looks of it. What are you running your Ubuntu environment on and how is your `/tmp` mounted?

**Comment by PeteBo** [comment]
@joshtrichards `/tmp` itself is not a mountpoint on its own but lives inside `/`
But inside `/tmp` there is the Collabora_Online.AppImage mounted to `/tmp/.mount_CollabyiK9Zz`

```
~# mount |grep tmp
udev on /dev type devtmpfs (rw,nosuid,relatime,size=15362596k,nr_inodes=3840649,mode=755,inode64)
tmpfs on /run type tmpfs (rw,nosuid,nodev,noexec,relatime,size=3079836k,mode=755,inode64)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,inode64)
tmpfs on /run/lock type tmpfs (rw,nosuid,nodev,noexec,relatime,size=5120k,inode64)
Collabora_Online.AppImage on /tmp/.mount_CollabyiK9Zz type fuse.Collabora_Online.AppImage (ro,nosuid,nodev,relatime,user_id=1001,group_id=1001)
tmpfs on /run/user/0 type tmpfs (rw,nosuid,nodev,relatime,size=3079836k,nr_inodes=769959,mode=700,inode64)
tmpfs on /run/user/1001 type tmpfs (rw,nosuid,nodev,relatime,size=3079836k,nr_inodes=769959,mode=700,uid=1001,gid=1001,inode64)

```

**Comment by joshtrichards** [comment]
@PeteBo Oh, right. You're using FUSE so I presume you're not running the AppImage in Docker like the original reporter is (I think).

**Comment by PeteBo** [comment]
@joshtrichards Correct, I'm not using docker here. Nextcloud is installed right in the VPS and inside Nextcloud I use the CODE App for Collabore Office.

But hold short, the OP says he also has the issue with the built-in CODE and he further explains that he can NOT reproduce it with NC AIO Image or Collabora Docker.

**Comment by jbofill** [comment]
I have this issue in two servers, one runs NC 27.1.5 and the other one NC 28.0.0
Both have built-in CODE Server 23.5.604

If you don't want to lose any content when writing to a new file, you must exit the editor and reopen the file again.
Otherwise the initial content is lost.

**Comment by with612** [comment]
Same issue here : 
 Collabora Online - Built-in CODE Server 23.5.604

used in NC 26.0.5

**Comment by les-les** [comment]
same issue with 
NC 28.0.1
and collabora 23.05.2.2 91c166d

**Comment by bugspencor** [comment]
same issue here

Ubuntu 22.04 (followed example installation document https://docs.nextcloud.com/server/latest/admin_manual/installation/example_ubuntu.html)
Nextcloud 27.1.5
Built-In Code Server  23.5.702
Nextcloud Office 8.2.4

**Comment by egypcio** [comment]
`+1`

### operating system

**Debian 12.4** (**bookworm**), **6.1.0-17-amd64** #1 SMP PREEMPT_DYNAMIC Debian **6.1.69-1** (**2023-12-30**) x86_64 GNU/Linux
- libfontconfig1:amd64    **2.14.1**-4
- libfuse2:amd64    **2.9.9**-6+b1
- php8.3    **8.3.2**-1+0-20240120.16+debian12-1.gbpb43448

hardening applied to `php.ini` and was removed while troubleshooting;

- cleared `disabled_functions`
- `memory_limit` set to 768M

while testing, played around a bit with fuse's settings

```# id www-data
uid=33(www-data) gid=33(www-data) groups=33(www-data),114(ssl-cert),1005(fuse)
```

```
$ grep -Ev '^#|^$' /etc/fuse.conf 
user_allow_other
mount_max = 800
```
might not even be related, but just in case: `kernel.unprivileged_userns_clone = 1` (debian's default config)

### nextcloud
version "**28.0.1.1**", standalone running in a virtual machine; using **built-in** collaboraonline.

```
"richdocuments": {
      "external_apps": "",
      "public_wopi_url": "https://nextcloud.localdomain",
      "types": "prevent_group_restriction",
      "canonical_webroot": "",
      "enabled": "yes",
      "disable_certificate_verification": "",
      "installed_version": "8.3.1",
      "wopi_url": "https://nextcloud.localdomain/apps/richdocumentscode/proxy.php?req="
    },
    "richdocumentscode": {
      "types": "",
      "enabled": "yes",
      "installed_version": "23.5.702"
    },
```
### coolwsd.log

[coolwsd.log](https://github.com/CollaboraOnline/richdocumentscode/files/13999000/coolwsd.log)

### Collabora_Online.AppImage (sha256, and version hash)
- sha256 = 7e134bbe1e06fb6e904842002effc2eea57fbcfa1fce1026b3053b1ed62fe9fa
- version-hash = f76e08a

```
# mount | grep -i colla
Collabora_Online.AppImage on /tmp/.mount_CollabKfWVfQ type fuse.Collabora_Online.AppImage (ro,nosuid,nodev,relatime,user_id=33,group_id=33)
```

#### tmp_dir (ini_get('upload_tmp_dir'))

```
# ls -laish /tmp/
ls: cannot access '/tmp/.mount_CollabKfWVfQ': Permission denied
total 4.0K
   1    0 drwxrwxrwt 29 root     root      580 Jan 20 19:02 .
   2 4.0K drwxr-xr-x 19 root     root     4.0K Jan 20 17:58 ..
1028    0 drwx------  3 www-data www-data   80 Jan 20 18:49 coolwsd.eaTrYTdix0
   5    0 drwxrwxrwt  2 root     root       40 Jan 20 18:02 .font-unix
   3    0 drwxrwxrwt  2 root     root       40 Jan 20 18:02 .ICE-unix
1095    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke4.tmp
1101    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke5.tmp
1104    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke6.tmp
1107    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke7.tmp
1108    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke8.tmp
1111    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oke9.tmp
1114    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okea.tmp
1117    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okeb.tmp
1120    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okec.tmp
1123    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121oked.tmp
1126    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okee.tmp
1127    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okef.tmp
1130    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okeg.tmp
1133    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okeh.tmp
1136    0 drwx------  2 www-data www-data   40 Jan 20 18:49 lu65121okei.tmp
   ?    ? d?????????  ? ?        ?           ?            ? .mount_CollabKfWVfQ
 952    0 drwx------  2 vinicius vinicius   60 Jan 20 18:42 ssh-XXXXvkCSUP
  14    0 drwx------  3 root     root       60 Jan 20 18:02 systemd-private-f4ed05d5ece045dd81df53d31ed2e7eb-redis-server.service-9fgHeO
  12    0 drwx------  3 root     root       60 Jan 20 18:02 systemd-private-f4ed05d5ece045dd81df53d31ed2e7eb-systemd-logind.service-RBDpDL
   6    0 drwx------  3 root     root       60 Jan 20 18:02 systemd-private-f4ed05d5ece045dd81df53d31ed2e7eb-systemd-timesyncd.service-2DEJ07
  91    0 drwx------  2 vinicius vinicius   60 Jan 20 18:02 tmux-1000
   2    0 drwxrwxrwt  2 root     root       40 Jan 20 18:02 .X11-unix
   4    0 drwxrwxrwt  2 root     root       40 Jan 20 18:02 .XIM-unix
```

### how to reproduce?

0. install the dependencies; do your thing and download the latest release of collabora ...
1. https://www.collaboraoffice.com/downloads/Collabora-Office-AppImage-Release/collabora-online-release-LATEST.AppImage
2. `chmox u+x collabora-online-release-LATEST.AppImage`
3. exec `collabora-online-release-LATEST.AppImage`
4. check `.mount_CollabXXXXX` under `tmp_dir`

**Comment by bugspencor** [comment]
As a workaround installed CODE from the deb-packages. That solved the issue on my two Nextcloud instances.

**Comment by egypcio** [comment]
> As a workaround installed CODE from the deb-packages. That solved the issue on my two Nextcloud instances.

funny that's exactly the move I had planned for today :laughing: 
... and I do confirm that running it as standalone server worked for me as well.

### packages

- code-brand    **23.05-27**
- coolwsd    **23.05.7.5-1**

one little thing to notice is that the collabora (richdocumentscode) got some code updates in the meantime; 2hours ago.

but what does worth mentioning is: by doing the standalone service, we no longer depend on the `AppImage` -- which may be the trouble maker for us (?)

**Comment by voggur** [comment]
I have the same problem.
installed: 27.1.4snap1
```
wsd-06667-06721 2024-02-07 09:30:04.143416 +0000 [ remotefontconfig_poll ] ERR  Remote config server has response status code: 404 (Not Found)| wsd/COOLWSD.cpp:1382
kit-07093-06690 2024-02-07 09:30:08.510876 +0000 [ kit_spare_003 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3065
wsd-06667-07092 2024-02-07 09:30:08.775541 +0000 [ docbroker_002 ] ERR  #-1: Serial mismatch 0 vs. 1| wsd/ProxyProtocol.cpp:168
wsd-06667-07092 2024-02-07 09:30:09.122254 +0000 [ docbroker_002 ] ERR  Expected to renamed the document [/tmp/coolwsd.2no3MGZ4sE/jails/6667-eb1c389b/wNowlQmeVccstiag/tmp/user/docs/wNowlQmeVccstiag/TEST3-TEST3.odt.upload] after template-loading to [/tmp/coolwsd.2no3MGZ4sE/jails/6667-eb1c389b/wNowlQmeVccstiag/tmp/user/docs/wNowlQmeVccstiag/TEST3-TEST3.odt.uploading] (ENOENT: No such file or directory)| wsd/DocumentBroker.cpp:1694
wsd-06667-07092 2024-02-07 09:30:25.506408 +0000 [ docbroker_002 ] ERR  Cannot save because CanSave::NoWriteSession though NeedToSave::Yes_Modified. May have data loss, but must stop| wsd/DocumentBroker.cpp:2451
wsd-06667-07092 2024-02-07 09:30:25.527846 +0000 [ docbroker_002 ] ERR  #40: Read failed, have 0 buffered bytes (ECONNRESET: Connection reset by peer)| net/Socket.hpp:1137
```

**Comment by dustbro** [comment]
Same here.
Nextcloud 29.0.0
Ubuntu 22.04

I've tried with built in CODE server, as well as installing deb from source. Text added to a newly created document doesn't save. editing an existing document is the only way to save data.

**Comment by hartsberger** [comment]
Same here. 
But noticed that it only affects documents, spreadsheets, and presentations. 
When creating a new diagram (.odg) the file is saved correctly.

**Comment by joho1968** [comment]
Same issue here.

Ubuntu **22.04.LTS**
Nextcloud **29.0.1**
Nextcloud Office app **8.4.2**
CODE **24.04.2.1 80a6f97**
_No docker, just a plain vanilla VPS_

If I create a document and start writing, nothing is saved. If I re-open the document and add text, it is saved 🤔

**Comment by hartsberger** [comment]
@Ashod, 
we are using the Nextcloud "Built-in CODE Server" ([App Store](https://apps.nextcloud.com/apps/richdocumentscode))
Can you tell me how to change the logging.level there?

But I can give you the messages I could find in ``/tmp/systemd-private-e3cff0cf9...bc8bd40-apache2.service-BflifT/tmp/coolwsd.lKV...uOv/coolwsd.log``

Create new document "test_autosave.docx" in Nextcloud: 
```
kit-1049852-1049852 2024-06-05 14:06:08.991377 +0200 [ kit_spare_016 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3046
wsd-2659978-1049850 2024-06-05 14:06:09.211605 +0200 [ docbroker_012 ] ERR  #-1: Serial mismatch 0 vs. 1| wsd/ProxyProtocol.cpp:168
wsd-2659978-2660020 2024-06-05 14:06:09.480083 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_searchprev.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.484943 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_searchnext.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.516395 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_statetablecellmenu.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.516694 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_prev.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.543917 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_next.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.551583 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_zoom.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.641410 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/lc_statetablecellmenu.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:06:09.644855 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/lc_zoom.svg].| wsd/FileServer.cpp:752
wsd-2659978-1049850 2024-06-05 14:06:09.902460 +0200 [ docbroker_012 ] ERR  Expected to renamed the document [/tmp/coolwsd.lKVdviRuOv/jails/2659978-6a6aa2e8/e4jdgoYlrIngZUxK/tmp/user/docs/wkEpgqmUEQ1SIbM1/test_autosave.docx.upload] after template-loading to [/tmp/coolwsd.lKVdviRuOv/jails/2659978-6a6aa2e8/e4jdgoYlrIngZUxK/tmp/user/docs/wkEpgqmUEQ1SIbM1/test_autosave.docx.uploading] (ENOENT: No such file or directory)| wsd/DocumentBroker.cpp:1839
```
--> I think the last line is the interesting one. 

Type in some text. 
Wait few seconds until the little dot from the save button disappears. 

Close the document: 
```
wsd-2659978-1049850 2024-06-05 14:07:47.347153 +0200 [ docbroker_012 ] ERR  #68: Read failed, have 0 buffered bytes (ECONNRESET: Connection reset by peer)| net/Socket.hpp:1146
```

Reopen the file and notice that it is empty: 
```
kit-1051984-1051984 2024-06-05 14:08:30.167519 +0200 [ kit_spare_017 ] WRN  Security warning: running without chroot jails is insecure.| kit/Kit.cpp:3046
wsd-2659978-2660020 2024-06-05 14:08:30.460414 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_searchprev.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.474537 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_searchnext.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.554314 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_statetablecellmenu.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.558035 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_prev.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.558199 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_next.svg].| wsd/FileServer.cpp:752
wsd-2659978-1051982 2024-06-05 14:08:30.575811 +0200 [ docbroker_013 ] ERR  #-1: Serial mismatch 0 vs. 1| wsd/ProxyProtocol.cpp:168
wsd-2659978-2660020 2024-06-05 14:08:30.592432 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/dark/lc_zoom.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.609851 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/lc_statetablecellmenu.svg].| wsd/FileServer.cpp:752
wsd-2659978-2660020 2024-06-05 14:08:30.613611 +0200 [ websrv_poll ] ERR  FileServerRequestHandler: File not found: Invalid URI request: [/browser/80a6f97/images/lc_zoom.svg].| wsd/FileServer.cpp:752
```

Type in some text again. 
Wait few seconds until the little dot from the save button disappears: 
```
wsd-2659978-1051982 2024-06-05 14:09:30.605607 +0200 [ docbroker_013 ] WRN  ToClient-164bc: client not consuming tiles; stalled for 10 seconds: removed tracking for 8 on the fly tiles| wsd/ClientSession.cpp:2452
```

Close the document: 
```
wsd-2659978-1051982 2024-06-05 14:09:49.054893 +0200 [ docbroker_013 ] ERR  #70: Read failed, have 0 buffered bytes (ECONNRESET: Connection reset by peer)| net/Socket.hpp:1146
```

Reopen the document again and notice that the text from the second try is still there.

**Comment by dustbro** [comment]
> For stand-alone installations, modify coolwsd.xml by setting the logging.level entry to trace, restart the service, reproduce the issue, copy the logs, compress, attach.

log attached:

[coolwsd.log](https://github.com/user-attachments/files/15587815/coolwsd.log)

**Comment by tobi-paul-helmut** [comment]
Same Issue here. Additionally I observed the following:
After Document (xlsx) creation I waited a few minutes. The issue didn't occur. Maybe the cron job running every 5min. on my machine does something to repair the issue.

Latest Nextcloud stable release
Built-in CODE Server

**Comment by timur-g** [comment]
There is a fix in the upcoming Built-in CODE.

**Comment by timar** [comment]
Fixed in version 24.4.402.

**Comment by dustbro** [comment]
confirmed. Fix works here. thanks!
